src/stores/root.ts

```diff
@@ -1404,6 +1404,21 @@ export class RootStore {
 					},
 				],
 			},
+			{
+				poolId: '703',
```

Comment:
FYI, 704 is otherwise identical but has more TVL. Used 704 in Frontier

Reply:
Oh ok

Huh, someone else made one too